### PR TITLE
Save indexed events in batches

### PR DESCRIPTION
**Pull request by dzhelezov**
This PR fixes 
- https://github.com/Joystream/hydra/issues/20
and partially
- https://github.com/Joystream/hydra/issues/21 : one can pass additinal entity references to `db/createDBConnection()`

### CODE REVIEW

File: substrate-query-framework/index-builder/src/db/helper.ts [hunk 1]
```diff
@@ -29,10 +29,12 @@ export function fillRequiredWarthogFields<T>(entity: DeepPartial<T>): DeepPartia
   return Object.assign(entity, requiredFields);
 }
 
-export async function createDBConnection(): Promise<Connection> {
+export async function createDBConnection(entities: any[] = []): Promise<Connection> {
```
Comment (metmirr) [inline substrate-query-framework/index-builder/src/db/helper.ts:32]: How do I pass warthog models here?

Comment (dzhelezov) [reply substrate-query-framework/index-builder/src/db/helper.ts:32]: Good point. I added `entities` field to `ProcessorOptions` which propagated down to here in `QueryNodeManager.process`. You can provide Warhog types to this field and they should be registered (but I haven't tested).

Comment (metmirr) [reply substrate-query-framework/index-builder/src/db/helper.ts:32]: 👍

**Review by metmirr — APPROVED** [review]
LGTM!

**Pull request merged**